corehq/apps/data_interfaces/templates/data_interfaces/edit_deduplication_rule.html

```diff
@@ -87,20 +89,22 @@
           {% crispy criteria_form %}
         </div>
       </div>
-      <fieldset {% if readonly %} disabled="disabled" {% endif %}>
-        <legend>{% trans "Actions" %}</legend>
-        <div data-bind="template: {name: 'update-case-property-action', foreach: propertiesToUpdate, as: 'propertyToUpdate'} "></div>
-        <div class="form-group">
-          <label class="control-label col-xs-12 col-sm-4 col-md-4 col-lg-2">
-            {% trans "Add Action" %}
-          </label>
-          <button class="btn btn-default" type="button" data-bind="click: addPropertyToUpdate">
-            <i class="fa fa-plus"></i>
-            {% trans "Add Update Case Property" %}
-          </button>
-          <input type="text" hidden name="properties_to_update" data-bind="value: serializedPropertiesToUpdate" />
-        </div>
-      </fieldset>
+      {% if allow_actions %}
+        <fieldset {% if readonly %} disabled="disabled" {% endif %}>
+          <legend>{% trans "Actions" %}</legend>
+          <div data-bind="template: {name: 'update-case-property-action', foreach: propertiesToUpdate, as: 'propertyToUpdate'} "></div>
+          <div class="form-group">
+            <label class="control-label col-xs-12 col-sm-4 col-md-4 col-lg-2">
+              {% trans "Add Action" %}
+            </label>
+            <button class="btn btn-default" type="button" data-bind="click: addPropertyToUpdate">
+              <i class="fa fa-plus"></i>
+              {% trans "Add Update Case Property" %}
+            </button>
+            <input type="text" hidden name="properties_to_update" data-bind="value: serializedPropertiesToUpdate" />
```

Comment:
generally for a `hidden` `input` tag, the valid `HTML` is `<input type="hidden" ... />`. `hidden` used as an attribute is not really considered valid HTML though some browsers may respect it. Curious if this was working for you during testing?

[see this discussion](https://www.feathery.io/blog/html-hidden-inputs#:~:text=An%20HTML%20hidden%20input%2C%20also,type%3D%E2%80%9Dhidden%E2%80%9D%3E.)

Reply:
Just for context, the change here is moving this existing code behind the `allow_actions` conditional -- that strange HTML was part of code that isn't changing beyond the indentation. I did verify that you can still save dedupe updates, which wouldn't work if the `properties_to_update` value wasn't being transmitted to the server. I am open to changing the HTML, but I think I'd prefer it to be part of a separate ticket. This PR was meant to be restricted to the flag